scripts/complexity.rb

```diff
@@ -0,0 +1,125 @@
+#! interpreter line
```

Comment:
I think we should add something like `#!/usr/bin/env ruby` instead of `#! interpreter line` here ☺️.